client/packages/invoices/src/Prescriptions/LineEditView/PrescriptionLineEditForm.tsx

```diff
@@ -423,6 +423,7 @@ export const PrescriptionLineEditForm: React.FC<
                   disabled={disabled}
                   onChange={e => {
                     updateNotes(e.target.value);
+                    setAbbreviation(''), setDefaultDirection('');
```

Comment:
woah, never seen this! kind of fun, but for convention am going to request we put them on separate lines 😁 
```suggestion
                    setAbbreviation('');
                    setDefaultDirection('');
```